package.json

```diff
@@ -44,7 +44,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run format; npm run lint; git add ."
+      "pre-push": "npm run lint"
```

Comment:
What do you think about keeping lint in `pre-commit`?
In this case, if lint fails - we won't have to do additional commit or amend

Reply:
Yeah good point.